meta-lmp-bsp/recipes-core/images/initramfs-ostree-lmp-recovery.bbappend

```diff
@@ -1 +1,32 @@
 FILESEXTRAPATHS:prepend := "${THISDIR}/${PN}:"
+
+# STM32MP1 specific defines
+PACKAGE_INSTALL:append:stm32mp1common = " \
```

Comment:
Technically this part doesn't need to be stm32mp1common specific.  But maybe we wait for another SoC before making this generic.

Reply:
I think I'll moved that when support for this recipe is added for another SoC